File: 백준/Bronze/5622. 다이얼/다이얼.py
arr = [['A','B','C'], ['D','E','F'],['G','H','I'],['J','K','L'],['M','N','O'],['P','Q','R','S'],['T','U','V'],['W','X','Y','Z']]

result = 2
lst = []
S = list(input())
for i in arr:
    result += 1
    for j in S:
        if j in i:
            lst.append(result)

print(sum(lst))

File: 백준/Bronze/5622. 다이얼/다이얼.js
const input = require("fs").readFileSync("/dev/stdin").toString().split("");

arr = [
  ["A", "B", "C"],
  ["D", "E", "F"],
  ["G", "H", "I"],
  ["J", "K", "L"],
  ["M", "N", "O"],
  ["P", "Q", "R", "S"],
  ["T", "U", "V"],
  ["W", "X", "Y", "Z"],
];
let result = [];
for (let i = 0; i < arr.length; i++) {
  for (let j = 0; j < input.length; j++) {
    if (arr[i].includes(input[j])) {
      result.push(i + 3);
    }
  }
}
console.log(result.reduce((a, b) => a + b, 0));
